### ISSUE TITLE
Example "n_clusters_hclust()" fails

### ISSUE THREAD

**Issue by strengejacke** [easystats/datawizard#353]
``` r
library("pvclust")
library(parameters)
x <- n_clusters_hclust(iris[1:4], iterations = 50, ci = 0.90)
#> Error in vapply(data, class, FUN.VALUE = character(1L)): values must be length 1,
#>  but FUN(X[[1]]) result is length 2
```

<sup>Created on 2023-01-17 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by strengejacke** [comment]
Seems to be a bug in `datawizard::transpose()`:

```r
load("test.RData")
datawizard::data_transpose(x)
```
[test.zip](https://github.com/easystats/datawizard/files/10434419/test.zip)

**Comment by strengejacke** [comment]
`parameters::n_clusters_hclust()` calls `datawizard::standardize(x, ...)`, so each column of `x` has multiple class attributes (`dw_transformer` and `numeric`), that's why `data_transpose()` fails.

**Comment by DominiqueMakowski** [comment]
Note that I recently encountered a related issue in using a dataset that went through `standardize()` with `lavaan`, when using `predict()` on the model. The error message was quite obscure `unknown dw_transformer` or something like that and I finally tracked it down to this added attribute by standardize, but I suppose it might create other issues

**Comment by strengejacke** [comment]
Yes, similar to the uncommon class attributes added by *haven*'s read-functions (`haven_labelled`), which may cause errors in different packages. We should probably revert this behaviour.

**Comment by strengejacke** [comment]
I just saw, we needed this when using `standardize()` etc. in model formulas, and then call `predict()`. Maybe that's the reason why `scale()` returns a matrix? :-D
